### PR TITLE
Add doc to support SiteOwnerManageLegacyServicePrincipalEnabled on Se…

### PR DESCRIPTION
**Pull request by RongqiZ**
Edit on the Set-SPOTenant command.

Add one parameter: SiteOwnerManageLegacyServicePrincipalEnabled

The ps command change is not in prod now.

### CODE REVIEW

**Review by dariomws — COMMENTED** [review]
@RongqiZ, will this parameter be included in the next release?

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:939]:
```suggestion
Allows or disallows the site owner to manage the Azure Access Control (ACS) service principal.

When the value is set to false, only tenant admins can create or update the ACS service principal from the admin center. If the value is set to True, both tenant admins and site owners can create or update the ACS service principal.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @RongqiZ, kindly commit the suggested changes. Many thanks.

**Review by get-itips — APPROVED** [review]

**Comment by RongqiZ** [comment]
> @RongqiZ, will this parameter be included in the next release?

This will release in weeks, I will merge this PR after the ps command parameter in 100% prod.

**Review by dariomws — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Comment by dariomws** [comment]
@RongqiZ, please let us know when it can be merged. Thanks!

**Comment by RongqiZ** [comment]
> @RongqiZ, please let us know when it can be merged. Thanks!

Hi @dariomws, the code merged on April 25, I checked the release process, it will be forked in mid May and release on end of May. We could merge this by end of this month.

**Comment by yogkumgit** [comment]
@RongqiZ Kindly suggest, can this be merged?

**Comment by yogkumgit** [comment]
@tiburd please investigate the status internally and take an action. Thanks

**Comment by RongqiZ** [comment]
> @RongqiZ Kindly suggest, can this be merged?

Hi we are waiting for the review results, and we planned to release this feature in Aug. We could merge this before that.

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md
Comment (RongqiZ) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md]: Updated

**Comment by RongqiZ** [comment]
Hi @yogkumgit @tiburd 
We are going to release the feature in the PR. Could you help to complete the PR for us? Many thanks!

**Comment by tiburd** [comment]
@RongqiZ
Hi,
If this PR is ready to merge, please remove "[Do Not Merge]" from the title and @ mention me again.
Thank you!

**Comment by RongqiZ** [comment]
Hi @tiburd 
Thanks for reminding! I removed the [Do Not Merge] in the title.

**Comment by VesaJuvonen** [comment]
Merging this to align with the conceptual docs which were just merged/updated and published.

Parameter has been in the SPO PowerShell cmdlets since the 16.0.23702.12000 release - so three months and flights are now rolling to production - so good to get this published as well.

**Pull request merged**
